app/views/comment_mailer/comment_added_notification.html.haml

```diff
@@ -1,27 +1,41 @@
-%table.full-width.header
-  %tr
-    %td.hero
-    %td
-      = image_tag "emails/icon-pencil-circle.png", class: "header-icon"
+- icon = "emails/icon-speechBubble-blue.png"
```

Comment:
super small detail, but going forward let's make sure we name assets in Snake case since we're in ruby land